src/_pytest/fixtures.py

```diff
@@ -1060,14 +1066,14 @@ def _parse_fixture_args(callable_or_scope, *args, **kwargs):
 
 
 def fixture(
-    callable_or_scope=None,
+    callable_or_scope: Union[Callable[..., Any], str] = None,
```

Comment:
this should be Optional[Union[...]] as it defaults to None (`strict_optional`)

Reply:
I think this is an invalid type annotation:
```
TypeError: Union[arg, ...]: each arg must be a type. Got Ellipsis.
```

Reply:
yeah I was typing that on my phone and was hoping you'd fill in what you already had :)